test/prompt/test_prompt_node.py

```diff
@@ -142,6 +144,21 @@ def test_create_prompt_model_dtype():
     assert model.model_name_or_path == "google/flan-t5-small"
 
 
+@pytest.mark.unit
```

Comment:
I'd argue whether this is really a unit test. It's basically testing the tokenizer. Let's mark it integration.
```suggestion
@pytest.mark.integration
```

A unit test would use a mock tokenizer. If you want to keep it unit, just mock the tokenizer instead of changing the marker.